### ISSUE TITLE
[User feedback]  The "last executed" field is copied when copying a rule.

### ISSUE THREAD

**Issue by EyeSeeTeaBotTest** [EyeSeeTea/metadata-synchronization-blessed#64]
## dhis2

Username: idelcano

## Browser
- Name: Mozilla
- Version: 5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36
- Platform: Linux x86_64

## User report
URL: https://extranet.who.int/dhis2-dev/api/apps/MetaData-Synchronization/index.html#/sync-rules/metadata

In this case "MAL - Update 22.06.20 - Policy Form - PROD" is a copy of "MAL - Update 22.06.20 - Policy Form"

"MAL - Update 22.06.20 - Policy Form - PROD" was never executed. 

![See screenshot here]( https://raw.githubusercontent.com/EyeSeeTeaBotTest/snapshots/master/screenshot-1592803904400423157.png )

**Comment by SferaDev** [comment]
```
    public replicate(): SyncRule {
        return this.updateName(`Copy of ${this.syncRule.name}`)
            .update({ lastExecuted: undefined })
            .updateId(generateUid());
    }
```

**Comment by adrianq** [comment]
Fixed in 2.0.0
